Assets/!/Prefabs/Interactble/Door.prefab

```diff
@@ -0,0 +1,154 @@
+%YAML 1.1
```

Comment:
Typo

Reply:
Otherwise lgtm.
we should also probably decide on a code structure, at the moment we are all over the place